### PR TITLE
Add tests for build column, combine cases, mean, compare

### PR DESCRIPTION
**Pull request by P-bibs**
Add tests for build column, combine cases, mean, and compare.

### CODE REVIEW

File: src/transformers/__tests__/buildColumn.test.ts [hunk 1]
```diff
@@ -20,3 +27,127 @@ test("throws error when new attribute collides with existing", () => {
...
+      )
+    ).records.map((row) => row.D)
+  ).toEqual(DATASET_A.records.map((row) => row.C));
+});
+
+test("build column does nothing to dataset with no records", async () => {
+  const COLLECTION_A_OUTPUT = cloneDataSet(EMPTY_RECORDS);
+  COLLECTION_A_OUTPUT.collections[0].attrs?.push({
+    description:
+      "An attribute whose values were computed with the formula A + B * F + 1",
```
Comment (thomascastleman) [inline src/transformers/__tests__/buildColumn.test.ts:65]: Oops I forgot to change this--Can you wrap these formulas in backticks and also make that change in `buildColumn.ts` (line 90)? I've been trying to wrap them in backticks but I missed this one.

File: src/transformers/__tests__/compare.test.ts [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+    expect(() =>
+      uncheckedNumericCompare(
+        DATASET_A,
+        "BadAttributeName",
+        "EvenBadderAttributeName"
+      )
+    ).toThrow("first");
+  });
+
+  // TODO: update this when attribute check changes are merged
```
Comment (thomascastleman) [inline src/transformers/__tests__/compare.test.ts:26]: Is this TODO done?

**Review by thomascastleman — APPROVED** [review]
Looks good!

File: src/transformers/__tests__/compare.test.ts (hunk 2 above)
Comment (P-bibs) [reply src/transformers/__tests__/compare.test.ts:26]: Yep, good catch

File: src/transformers/__tests__/buildColumn.test.ts (hunk 1 above)
Comment (P-bibs) [reply src/transformers/__tests__/buildColumn.test.ts]: Pushed in recent commit. Everything look good?

Comment (thomascastleman) [reply src/transformers/__tests__/buildColumn.test.ts]: Nice looks good!

**Pull request merged**